### PR TITLE
Issue #5924  force httpclient version in asciidoctor plugin 

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: documentation/jetty-documentation/pom.xml [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
+          <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.13</version>
+          </dependency>
+          <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.13</version>
+          </dependency>
```
Comment (sbordet) [inline documentation/jetty-documentation/pom.xml:39]: Why would javadoc work if this is added?

**Review by sbordet — CHANGES_REQUESTED** [review]
Sorry there are no commit comments, so this seems like a random fix.
-1 unless explicitly details why would fix javadocs.

**Comment by sbordet** [comment]
After analysis with @olamy, this is the only viable workaround for now, see 2f766632b3462b8393e40e5f6ed5d32ed7ac577c.

**Comment by olamy** [comment]
replaced by 2f766632b3462b8393e40e5f6ed5d32ed7ac577c

**Pull request closed without merging**